homeassistant/components/deconz/sensor.py

```diff
@@ -268,7 +268,10 @@ def async_add_sensor(_: EventType, sensor_id: str) -> None:
                 continue
 
             no_sensor_data = False
-            if description.value_fn(sensor) is None:
+            try:
+                if description.value_fn(sensor) is None:
```

Comment:
It looks like the error was 
```
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
TypeError: float() argument must be a string or a real number, not 'NoneType'
```

Can you check if `sensor is None` instead of using a `try` block?

Reply:
I started doing a more detailed check but considered this more concise. Let me whip that up

Reply:
Better?

Reply:
LGTM 👍